test/powershell/Language/Scripting/Generics.Tests.ps1

```diff
@@ -0,0 +1,95 @@
+﻿Describe "Generics support" -Tags "CI" {
+    #
+    # 3 types are tested, list, stack, and dictionary.
+    #
+    # list and stack are in different assemblies, and dictionary
+    # takes more than one type parameter.
+    #
+    $listType = "system.collections.generic.list"
+    $stackType = "system.collections.generic.stack"
+    $dictionaryType = "system.collections.generic.dictionary"
+
+    It 'list[Int]' {
+        $x = [system.collections.generic.list[int]]::New()
+        $x.Add(42)
+        $x.count | Should Be 1
```

Comment:
We should test more than 1 item in the collection because non-collections treat count specially, e.g.
```powershell
PS> (42).Count -eq 1
True
```